serde_arrow/src/internal/serialization_ng/time64_builder.rs

```diff
@@ -56,14 +56,14 @@ impl SimpleSerializer for Time64Builder {
         let timestamp = {
             use chrono::naive::NaiveTime;
             let time = v.parse::<NaiveTime>()?;
-            time.num_seconds_from_midnight() as u64 * 1_000_000_000u64 + time.nanosecond() as u64
+            time.num_seconds_from_midnight() as i64 * 1_000_000_000i64 + time.nanosecond() as i64
```

Comment:
I think here is still a bug right: if I use the `Time64(Microsecond)` type this will result in the wrong timestamps

Reply:
I commented out this whole function as I think it makes sense to include deserialization as well. And then to include round trip tests from strings to arrays and back.